docs/api/api-reference.md

```diff
@@ -9,5 +9,22 @@ contentType: reference
 ---
 
 
-<redoc theme='{"typography": {"font-family":"Open sans", "headings":{"fontFamily":"Open sans"}}}' spec-url="/api/v1/openapi.yml"></redoc>
-<script src="https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.js"> </script>
+<redoc
+  spec-url="/api/v1/openapi.yml"
+  disable-search
```

Comment:
No strong feelings really, but curious why we decided to disable this.

Reply:
I've tried using it in the past, and the UX isn't great. 
I prefer no search to bad search, especially when the list isn't that long, and people can just use their browser search.

Reply:
Ah, gotcha. I think that's fair. Thanks for the explanation :+1: